citation.tsv

```diff
@@ -73,7 +73,7 @@
 	seriesName	Name	The name of the dataset series		text	69	#VALUE	TRUE	FALSE	FALSE	TRUE	FALSE	FALSE	series	citation	
 	seriesInformation	Information	Can include 1) a history of the series and 2) a summary of features that apply to the series		textbox	70	#VALUE	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	series	citation	
 	software	Software	Information about the software used to generate the Dataset		none	71	,	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		citation	https://www.w3.org/TR/prov-o/#wasGeneratedBy
-	softwareName	Name	The name of software used to generate the Dataset		text	72	#VALUE	FALSE	TRUE	FALSE	FALSE	FALSE	FALSE	software	citation	
+	softwareName	Name	The name of software used to generate the Dataset		text	72	#VALUE	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	software	citation
```

Comment:
I would not delete the "tab" at the end of the line, since there is another column (termURI).

Reply:
Das hat upstream lustigerweise zw 6.1 und 6.2 entfernt

Reply:
Möglichst weniger Änderungen zu Upstream vs Konsistenz - was gewinnt bei dir?